### DISCUSSION TITLE
Adding video/embedded video to the product gallery

### DISCUSSION THREAD

**Discussion by SalahAdDin** [medusajs/medusa#7996] [Feature Requests]
Some of the best e-commerce websites enable sellers to add videos to their product galleries, as an example, you can see this tutorial to add them to WooCommerce: https://iconicwp.com/blog/add-featured-video-woocommerce-product-gallery/

Can we have it as either a feature or a plugin?

Thank you!

**Comment by Gumbraise** [comment]
Anyone?

**Reply by SalahAdDin** [reply]
Not an answer yet.
